cottoncandy/interfaces.py

```diff
@@ -212,6 +212,29 @@ def get_bucket_objects(self, **kwargs):
         response = request.all()
         return response
 
+    def get_bucket_size(self, limit=10000000, page_size=10000000):
+        '''
```

Comment:
for now, lets stick with numpydoc style to be consistent with the rest of the code base.